assets/js/modules/analytics-4/datastore/settings.js

```diff
@@ -335,7 +347,7 @@ export function validateCanSubmitChanges( select ) {
 }
 
 export function validateHaveSettingsChanged( select, state, keys ) {
-	const { settings, savedSettings } = state;
+	const { settings: newSettings, savedSettings } = state;
```

Comment:
Let's keep this function as before as these references to state are likely in other related functions as well. I'm guessing this was updated earlier on in the PR but isn't changed anymore?